src/layout/css/_about.scss

```diff
@@ -43,38 +43,38 @@ section p {
   font-size: 15px;
 }
 
-.left-overlay_about_content {
+.ts-gradient-background-start {
```

Comment:
I prefixed these classes with `ts-` to make it clear that they come from us. I also tried to align the name with the naming scheme used by Bootstrap,